exes/mqtt-broadcaster/CMakeLists.txt

```diff
@@ -25,29 +30,29 @@ target_link_libraries(mqtt-broadcaster
     tfc::sparkplug::proto
 )
 
-target_compile_definitions(mqtt-broadcaster
+target_compile_definitions(mqtt_broadcaster
   PUBLIC
     ASYNC_MQTT_USE_TLS
 )
 
-target_compile_options(mqtt-broadcaster PRIVATE -Wno-exit-time-destructors)
+target_compile_options(mqtt_broadcaster PRIVATE -Wno-exit-time-destructors)
 
-if (BUILD_TESTING)
-  add_subdirectory(tests)
-endif ()
+# if (BUILD_TESTING)
```

Comment:
remove comments, why is this needed?

Reply:
this was used to add tests in the pipelines when needed, but the tests aren't ready so taking this out.